src/main/java/gigaherz/toolbelt/slot/ExtensionSlotBelt.java

```diff
@@ -186,7 +189,7 @@ public void playerClone(PlayerEvent.Clone event)
             ExtensionSlotBelt newBelt = get(newPlayer);
             if (oldBelt == null) return;
             ItemStack item = oldBelt.getBelt().getContents();
-            if (newBelt == null) {
+            if (newBelt == null && item != null) {
```

Comment:
Same as before, getContents should never ever be returning null, so that needs to be fixed instead, this change shouldn't be necessary.